### ISSUE TITLE
Slide listner

### ISSUE THREAD

**Issue by ArslanYasin** [denzcoskun/ImageSlideshow#39]
Hello,
Thank you for the library. I would like to ask is there any listener on change image or slide?

Thanks in advance

**Comment by denzcoskun** [comment]
Hello,

You can with new version.

```Kotlin
imageSlider.setItemChangeListener(object : ItemChangeListener {
     override fun onItemChanged(position: Int) {
         // You can listen here.
    }
})
```

```
implementation 'com.github.denzcoskun:ImageSlideshow:0.1.0'
```

**Comment by bitactro** [comment]
Its not working .. I am using java.. everything is working fine expect listeners. Do you know what could be the cause?

**Comment by enriqif** [comment]
> Hello,
> 
> You can with new version.
> 
> ```kotlin
> imageSlider.setItemChangeListener(object : ItemChangeListener {
>      override fun onItemChanged(position: Int) {
>          // You can listen here.
>     }
> })
> ```
> 
> ```
> implementation 'com.github.denzcoskun:ImageSlideshow:0.1.0'
> ```

Hello!!

This listener works perfect! but what about if I want to stop this listener? 

When I change the fragment setItemChangeListener continues executing

I try using imageSlider.stopSliding() and startSliding() in onPause() and onResume() methods, but this works just when I close the app 

Can you give some idea to solve this?

Thanks!

**Comment by denzcoskun** [comment]
Hello, thank you for your nice comments.
Maybe you can define a boolean variable and update it in onResume and onPause so you can check this boolean in listener.

**Comment by enriqif** [comment]
> Hello, thank you for your nice comments.
> Maybe you can define a boolean variable and update it in onResume and onPause so you can check this boolean in listener.

Yeah.. I did it.. but as I said before, the listener stop and resume in those methods only if I close the app, but when I load other fragment setItemChangeListener continues executing and I don't need that.

I don't know which lifecycle's method should I use to achive this.
